Automattic/wp-calypso
# auth: remove oauth flux implementation

#### Changes proposed in this Pull Request

* Remove the `lib/oauth-store` flux store but keep helper functions as `lib/oauth-utils`
* Move state previously managed with flux to `auth/login.jsx` internal component state

#### Testing instructions

* Build the desktop app locally and make sure login (with 2fa!) is still working
* Take a closer look at the form state (e.g. disabled inputs when form is submitting)

fixes #24145
